src/tooltipServiceWrapper.ts

```diff
@@ -153,39 +122,38 @@ class TooltipServiceWrapper implements ITooltipServiceWrapper {
         let touchEndEventName: string = TooltipServiceWrapper.touchEndEventName();
         let isPointerEvent: boolean = TooltipServiceWrapper.usePointerEvents();
 
-        selection.on(touchStartEventName + ".tooltip", () => {
+        selection.on(touchStartEventName + '.tooltip', () => {
             this.visualHostTooltipService.hide({
-                immediately: true,
                 isTouchEvent: true,
+                immediately: true,
             });
+        });
 
             let tooltipEventArgs = this.makeTooltipEventArgs<T>(rootNode, isPointerEvent, true);
-            if (!tooltipEventArgs) {
+            if (!tooltipEventArgs)
                 return;
-            }
+
```

Comment:
This is not recommend, please always use curly brackets

Reply:
fixed